app/helpers/react_helper.rb

```diff
@@ -216,7 +216,6 @@ def govbox_set_data(search)
     if show_results_format?(affiliate) && !affiliate.display_created_date_on_search_results?
       reject_keys_from_hash(govbox_set_json[:federalRegisterDocuments], 'publication_date') if govbox_set_json[:federalRegisterDocuments].present?
       reject_keys_from_hash(govbox_set_json[:newsItems], 'published_at') if govbox_set_json[:newsItems].present?
-      reject_keys_from_hash(govbox_set_json[:youtubeNewsItems], 'published_at') if govbox_set_json[:youtubeNewsItems].present?
```

Comment:
I'm not sure why this line was added in the first place, but it's stripping off the `published_at` date for YouTube results on the main search page, causing it to default to `a few seconds ago`.